### ISSUE TITLE
Repeated calls to AddTab, AddModules can erase file content

### ISSUE THREAD

**Issue by kwathen** [BioPharmSoftGrp/PREP#87]
If you call the following
AddTab( "MyNewTab" )

Make changes in the UI or Server file.  Save your work.

Call this again
AddTab( "MyNewTab" ) 

The changes that were made are overwritten and back to default.   

It looks like we need to check if a file exists before writting to it.   Originally, it was looking and if a file existed it would add add a 1 to the file name.    For example, when the second call to AddTab occurs the mod_MyNewTabUI.R exists so try mod_MyNewTabUI1.R and if it exists then mod_MyNewTabUI2.R  and so on.  

Basically I think this will only happen by accident but don't want to erase work as it is not recoverable.
